src/color/ui.json

```diff
@@ -97,6 +146,16 @@
             "public": true,
             "onLight": true,
             "usage": "Default hover affordance for background colors",
+            "deprecated": true,
+            "comment": "Deprecated token",
+            "reference": "color-ui-bkg-hover  -default",
+            "version": "3.15.0"
+          },
+          "default": {
+            "value": "0000000f",
```

Comment:
Is ths supposed to be a hex value?

Reply:
It's an 8 digit hex. https://www.quackit.com/css/color/values/css_hex_color_notation_8_digits.cfm#:~:text=Eight%2Ddigit%20hex%20notation%20consists,alpha%20chanel%20of%20the%20color.&text=The%20RR%20represents%20the%20red,GG%20represents%20the%20green%20component.

Reply:
Makes sense, just wanted to make sure it wasn't a copy paste error.